### ISSUE TITLE
API: Missed Configuration-Version header in version 2.4

### ISSUE THREAD

**Issue by balin777** [haproxytech/dataplaneapi#239]
Dataplane API not does not return title in all versions older than 2.3.9
![2 3 9](https://user-images.githubusercontent.com/41673679/153182269-2f2586ff-b126-4905-b5c4-a8b98dbe772a.PNG)
![2 4 2](https://user-images.githubusercontent.com/41673679/153182274-05f57f49-a8ae-49ec-ab37-2e245303bf81.PNG)

**Comment by matheuscaselani** [comment]
Same problem on 2.4:latest and 2.2.20. Any help ? D:

**Comment by mjuraga** [comment]
Yes we had a fix in the newer version which fixes that. All configuration endpoints now return Configuration-Version header on all requests.
